packages/cli-platform-apple/src/commands/buildCommand/createBuild.ts

```diff
@@ -36,6 +36,7 @@ const createBuild =
 
     let {xcodeProject, sourceDir} = getXcodeProjectAndDir(
       platform,
+      platformName,
```

Comment:
At this point it would be good to rename “platform” to “platformConfig”

Reply:
Renamed